Chapter4/main.cpp

```diff
@@ -573,7 +575,7 @@ class OpenXRTutorial {
         m_indexBuffer = m_graphicsAPI->CreateBuffer({GraphicsAPI::BufferCreateInfo::Type::INDEX, sizeof(uint32_t), sizeof(cubeIndices), &cubeIndices});
 
         // XR_DOCS_TAG_BEGIN_Update_numberOfCuboids
-        size_t numberOfCuboids = 64 + 2 + 2;
```

Comment:
looks like this line is included in the generated tutorial HTML, does it still make sense with the change?

Not sure if the text needs editing. 

![image](https://github.com/user-attachments/assets/93b13691-aad5-473c-bae4-ca19ee7a2c90)

Reply:
Good point. I've updated the tutorial text.